glass/grf/_transformations.py

```diff
@@ -8,7 +8,7 @@
 
 from numpy.typing import NDArray  # noqa: TC002
 
-from glass.grf._core import corr, dcorr, icorr
+from glass.grf import corr, dcorr, icorr
```

Comment:
This one has to remain `from glass.grf` syntax as otherwise we run into a circular import